src/System.Security.Cryptography.Pkcs/tests/EnvelopedCms/DecryptTests.cs

```diff
@@ -21,7 +21,9 @@ namespace System.Security.Cryptography.Pkcs.EnvelopedCmsTests.Tests
 {
     public static partial class DecryptTests
     {
-        [Fact]
+        public static bool IsNet462OrNewer() => (!PlatformDetection.IsFullFramework || PlatformDetection.IsNetfx462OrNewer());
```

Comment:
Oh, and this could be a property that gets set just the once:

```C#
public static bool SupportsCngCertificates { get; } = (!PlatformDetection.IsFullFramework || PlatformDetection.IsNetfx462OrNewer());